src/com/pentalog/pentastagiu/homework2/PR2/Basket.java

```diff
@@ -0,0 +1,33 @@
+package com.pentalog.pentastagiu.homework2.PR2;
+
+public class Basket {
+    private int BasketId;
+    private Product produs;
```

Comment:
aici ar fi mai potrivita o lista de produse, pt ca in fiecare cos de cumparaturi ai unul sau mai multe produse